compiler-plugin/src/main/java/io/ballerina/stdlib/kafka/plugin/PluginUtils.java

```diff
@@ -73,4 +75,18 @@ public static boolean validateModuleId(ModuleSymbol moduleSymbol) {
         }
         return false;
     }
+
+    public static boolean isWithinRange(LineRange lineRange, LinePosition pos) {
```

Comment:
Shall we move this function to the `compiler-plugin-tests` folder as it is only used there?

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/362/commits/f31688579f94275e6fbb56ae379e0aa5b7e43c56